Feat/operator-case

kycheng opened the pull request:
- Added commit-test-image configuration to integration-test.yaml for upstream repository handling.
- Updated feature files to use dynamic namespace names for various deployment scenarios.
- Introduced new operator.feature for testing the deployment of Harbor instances via the operator.
- Adjusted existing feature files to reflect changes in namespace naming conventions.

nanjingfm commented on .tekton/integration-test.yaml:
```diff
@@ -43,6 +43,15 @@ spec:
       value:
         image-repository: build-harbor.alauda.cn/devops/harbor-ce-test
         dockerfile-path: Dockerfile
+    - name: commit-test-image
```
chart 和 operator 分支策略不一样，不能直接更新吧

kycheng committed:
feat: update integration test configuration and add operator deployment tests

    - Added commit-test-image configuration to integration-test.yaml for upstream repository handling.
    - Updated feature files to use dynamic namespace names for various deployment scenarios.
    - Introduced new operator.feature for testing the deployment of Harbor instances via the operator.
    - Adjusted existing feature files to reflect changes in namespace naming conventions.
